roles/sft-server/files/nginx.conf

```diff
@@ -0,0 +1,34 @@
+
+user  nginx;
+worker_processes  1;
```

Comment:
Why do you restrict this to one worker? Why not setting this to `auto`, so multiple CPU cores can be used?

Reply:
Fixed.